registry.json

```diff
@@ -3529,5 +3529,19 @@
     "telegram": "woltere",
     "operatorName": "Wolter",
     "github": "RV64-Stake-Pool"
+  },
+  {
+    "ticker": "SMAUG",
+    "poolId": "abacadaba9f12a8b5382fc370e4e7e69421fb59831bb4ecca3a11d9b",
+    "social": {
+      "website": "https://smaug.pool.pm",
+      "twitter": "SmaugPool",
+      "facebook": "",
+      "instagram": "smaugpool",
+      "youtube": "https://www.youtube.com/channel/UC9ZapTnuEs1lNFF4CiZGVfA"
+    },
+    "telegram": "SmaugPool",
+    "operatorName": "Smaug",
```

Comment:
Please can you replace this with your actual name or leave black?

Reply:
Sure, one sec.

Reply:
Done.